src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketCredentials.java

```diff
@@ -53,26 +55,34 @@ private BitbucketCredentials() {
         throw new IllegalAccessError("Utility class");
     }
 
+    /**
+     * Performs a lookup of credentials for the given context. Additionally, usage of the credentials is tracked for the
+     * given {@link SCMSourceOwner} via {@link CredentialsProvider#track(Item, Credentials)}
+     */
     @CheckForNull
     static <T extends StandardCredentials> T lookupCredentials(@CheckForNull String serverUrl,
```

Comment:
As I already quickly mentioned I'd personally even go for renaming the method here to something like `lookupCredentialsAndTrackUsage` but I'll await feedback on that.

https://github.com/jenkinsci/bitbucket-branch-source-plugin/pull/834#discussion_r1542517069